environment/vm/lima/yaml.go

```diff
@@ -167,6 +171,7 @@ type Config struct {
 	Memory       string            `yaml:"memory,omitempty"`
 	Disk         string            `yaml:"disk,omitempty"`
 	Mounts       []Mount           `yaml:"mounts,omitempty"`
+	MountType    string            `yaml:"mountType,omitempty"`
```

Comment:
The YAML type definition is still under discussion (https://github.com/lima-vm/lima/pull/726)